dashboard/app/models/census/census_summary.rb

```diff
@@ -54,41 +54,37 @@ def self.seed_all
   end
 
   # Loads/merges the data from a CSV into the table.
-  # Requires a block to parse the row.
   # @param filename [String] The CSV file name.
   # @param options [Hash] The CSV file parsing options.
   # @param is_dry_run [Boolean] Allows testing of importing a CSV by rolling back any changes.
   def self.merge_from_csv(filename, options = CSV_IMPORT_OPTIONS, is_dry_run: false)
-    census_summaries = nil
     entries_skipped = 0
 
-    ActiveRecord::Base.transaction do
-      census_summaries = CSV.read(filename, **options).each do |row|
-        parsed = row.to_hash.symbolize_keys
+    census_summaries = CSV.read(filename, **options).each do |row|
```

Comment:
sorry but can you please split the changes to this file out into a separate PR? the cronjob and the seed change both have the potential to have unforeseen consequences, and it will be easiest to resolve if we have two separate PRs to work with so we can revert one easily.

Reply:
https://github.com/code-dot-org/code-dot-org/pull/54268